src/domains/citizen-auth-common/03_storage.tf

```diff
@@ -142,6 +146,112 @@ resource "azurerm_storage_container" "lv_audit_logs_storage_logs" {
   container_access_type = "private"
 }
 
+
+
+###
+# Immutable LV Audit Log Storage
+###
+module "immutable_lv_audit_logs_storage" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3//storage_account?ref=v7.32.1"
+
+  name                          = replace(format("%s-imm-lv-logs-st", local.product), "-", "")
```

Comment:
Maybe for better readability we should put `immutable` tag at the end of the storage name
es. `"%s-lv-logs-imm-st"`
So we have `[project]-[context]-[options]-[type]`

Reply:
Done in [fix based on comments](https://github.com/pagopa/io-infra/pull/778/commits/1e4268778c5a0b35cfe764917907553e84ffb6de)